doc/source/whatsnew/v0.18.2.txt

```diff
@@ -94,6 +94,9 @@ Other enhancements
 - ``eval``'s upcasting rules for ``float32`` types have been updated to be more consistent with NumPy's rules.  New behavior will not upcast to ``float64`` if you multiply a pandas ``float32`` object by a scalar float64. (:issue:`12388`)
 - ``Series`` has gained the properties ``.is_monotonic``, ``.is_monotonic_increasing``, ``.is_monotonic_decreasing``, similar to ``Index`` (:issue:`13336`)
 
+- ``pd.read_json`` has gained support for reading json lines with ``lines`` option (:issue:`9180`)
+- ``pd.read_json`` has gained support for accepting encoding of file or bytes buffer with ``encoding`` option (:issue:`13356`)
```

Comment:
actually, pls review the json docs (doc/source/io.rst) and see if anything needs to be mentioned / added (e.g. maybe a 1-sentence about how can handle line delimited json)